### DISCUSSION TITLE
Webpack 5 not pulling in my node_modules .scss file

### DISCUSSION THREAD

**Discussion by maloomis** [webpack/webpack#16716] [Q&A]
# Bug report

**What is the current behavior?**
I am getting an error when trying to import my .scss file located in node_modules

**If the current behavior is a bug, please provide the steps to reproduce.**

Package.json
`{
  "name": "app",
  "version": "0.1.0",
  "private": true,
  "sideEffects": [
    "*.scss"
  ],
  "scripts": {
    "test": "jest",
    "start": "node -r esm dist/server/index.js",
    "start-dev": "npm-run-all --parallel watch:server watch:client",
    "watch:server": "nodemon",
    "watch:client": "webpack --watch --progress --mode=development",
    "build": "jest && rm -rf dist && yarn build-server && yarn build-client",
    "build-server": "tsc -sourcemap -p ./server/tsconfig.json",
    "build-client": "webpack",
    "sandbox-setups": "yarn sandbox-setup:admin-checklist && yarn sandbox-setup:welcome-email && yarn sandbox-setup:home-welcome-message && yarn sandbox-setup:onboarding-tasks",
    "sandbox-setup:admin-checklist": "yarn --cwd ./src/sandbox/admin-checklist",
    "sandbox-setup:welcome-email": "yarn --cwd ./src/sandbox/welcome-email",
    "sandbox-setup:home-welcome-message": "yarn --cwd ./src/sandbox/home-welcome-message",
    "sandbox-setup:onboarding-tasks": "yarn --cwd ./src/sandbox/onboarding-tasks"
  },
  "dependencies": {
    "@pluralsight/analytics-facade": "4.10.0-beta.2",
    "@pluralsight/design-tokens": "0.3.1",
    "@pluralsight/headless-styles": "0.1.2",
    "@pluralsight/icons": "0.1.2",
    "@pluralsight/ps-cdk": "4.7.2",
    "@pluralsight/ps-design-system-banner": "11.1.1",
    "@pluralsight/ps-design-system-breadcrumb": "12.1.1",
    "@pluralsight/ps-design-system-button": "24.1.1",
    "@pluralsight/ps-design-system-card": "19.1.1",
    "@pluralsight/ps-design-system-core": "10.0.4",
    "@pluralsight/ps-design-system-dialog": "15.0.11",
    "@pluralsight/ps-design-system-icon": "25.4.0",
    "@pluralsight/ps-design-system-linearprogress": "10.0.14",
    "@pluralsight/ps-design-system-link": "13.0.11",
    "@pluralsight/ps-design-system-normalize": "7.0.4",
    "@pluralsight/ps-design-system-scrollable": "6.0.11",
    "@pluralsight/ps-design-system-textarea": "10.1.1",
    "@pluralsight/ps-design-system-theme": "8.1.24",
    "@pluralsight/ps-design-system-util": "10.1.3",
    "@pluralsight/ps-design-system-verticaltabs": "8.1.1",
    "@pluralsight/ps-express-health": "2.0.1",
    "@pluralsight/ps-session-node": "1.4.0",
    "@pluralsight/ps-vault-kubernetes": "2.0.2",
    "@pluralsight/ps-web-logging": "1.1.1",
    "@pluralsight/react-channels-list": "8.2.8",
    "@pluralsight/welcome-email-widget": "0.0.118",
    "async-exit-hook": "2.0.1",
    "aws-cdk-lib": "2.56.0",
    "aws-sdk": "2.1280.0",
    "bunyan": "1.8.15",
    "connect-requestid": "1.1.0",
    "cookie-parser": "1.4.6",
    "dotenv": "16.0.1",
    "ejs": "3.1.8",
    "esm": "3.2.25",
    "express": "4.18.1",
    "knex": "2.3.0",
    "lottie-react": "2.3.1",
    "newrelic": "8.14.1",
    "qs": "6.10.3",
    "rambda": "7.4.0",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-router-dom": "6.3.0",
    "sass": "1.51.0",
    "ts-node": "10.7.0",
    "typescript": "4.7.2"
  },
  "devDependencies": {
    "@babel/core": "7.17.12",
    "@babel/plugin-transform-modules-commonjs": "7.18.6",
    "@babel/plugin-transform-react-jsx": "7.17.12",
    "@babel/preset-env": "7.17.12",
    "@babel/preset-react": "7.17.12",
    "@babel/preset-typescript": "7.18.6",
    "@reduxjs/toolkit": "1.8.3",
    "@testing-library/jest-dom": "5.16.4",
    "@testing-library/react": "13.2.0",
    "@testing-library/user-event": "14.2.0",
    "@types/bunyan": "1.8.8",
    "@types/express": "4.17.13",
    "@types/jest": "28.1.6",
    "@types/node": "17.0.35",
    "@types/react": "18.0.9",
    "@types/react-dom": "18.0.5",
    "@types/react-transition-group": "4.4.5",
    "babel-jest": "28.1.3",
    "babel-loader": "8.2.5",
    "babel-plugin-react-require": "3.1.3",
    "colors": "1.4.0",
    "connected-react-router": "6.9.3",
    "constructs": "10.1.209",
    "css-loader": "6.7.1",
    "file-loader": "6.2.0",
    "glamor": "2.20.40",
    "html-webpack-plugin": "5.5.0",
    "jest": "28.1.3",
    "jest-environment-jsdom": "28.1.3",
    "launchdarkly-node-server-sdk": "6.4.2",
    "nodemon": "2.0.16",
    "npm-run-all": "4.1.5",
    "ramda": "0.28.0",
    "react-redux": "8.0.2",
    "react-transition-group": "4.4.2",
    "redux": "4.2.0",
    "redux-mock-store": "1.5.4",
    "redux-thunk": "2.4.1",
    "sass-loader": "13.0.0",
    "speed-measure-webpack-plugin": "1.5.0",
    "style-loader": "3.3.1",
    "styled-components": "5.3.5",
    "ts-jest": "28.0.7",
    "ts-loader": "9.3.0",
    "uuid": "8.3.2",
    "webpack": "5.75.0",
    "webpack-cli": "4.9.2"
  },
  "engines": {
    "node": "16.17.0"
  }
}
`

Webpack.config
`import path from 'path'
import SpeedMeasurePlugin from 'speed-measure-webpack-plugin'

console.log(path.join(__dirname, './node_modules'))

//const speedMeasurePlugin = new SpeedMeasurePlugin()

import CompileReporter from './lib/compile-reporter'
CompileReporter()

export default //speedMeasurePlugin.wrap(
{
  mode: 'production',
  externals: [],
  stats: {
    loggingDebug: ['sass-loader'],
  },
  externalsPresets: {
    node: true,
  },
  entry: {
    app: './src/index.tsx',
  },
  output: {
    path: path.join(__dirname, '/dist/client'),
    filename: '[name].bundle.js',
  },
  module: {
    rules: [
      {
        test: /\.(js|jsx)$/,
        include: path.resolve(__dirname, 'src'),
        exclude: /node_modules/,
        loader: 'babel-loader',
      },
      {
        test: /\.(ts|tsx)$/,
        include: path.resolve(__dirname, 'src'),
        exclude: /node_modules/,
        use: 'ts-loader',
      },
      {
        test: /\.(scss|css)$/,
        use: ['style-loader', 'css-loader', 'sass-loader'],
        include: path.resolve(__dirname, 'src'),
      },
      {
        test: /\.(scss|css)$/,
        use: ['style-loader', 'css-loader', 'sass-loader'],
        include: path.resolve(__dirname, 'node_modules'),
      },
      {
        test: /\.svg$/,
        use: [
          {
            loader: 'svg-url-loader',
            options: {
              limit: 10000,
            },
          },
        ],
      },
      {
        test: /\.(png|jpe?g|gif)$/i,
        use: [{ loader: 'file-loader' }],
      },
    ],
  },
  resolve: {
    extensions: ['.tsx', '.ts', '.js'],
  },
  devtool: 'source-map',
}`

index.scss
`@use '@pluralsight/design-tokens/npm/scss/_variables.scss' as *;

// @import '@pluralsight/ps-design-system-normalize';

/* This CSS is common to all pages */

// body {
//   background-color: var(--ps-colors-background-dark-1);
//   color: var(--ps-colors-text-icon-high-on-dark);
// }

// a {
//   color: var(--ps-colors-primary-action-text-dark-theme);
// }`

**What is the expected behavior?**

The .scss file is imported and we are able to use the color tokens

**Other relevant information:**
webpack version:  5.75.0
Node.js version: 16.17.0
Operating System: Mac
Additional tools:

**Comment by alexander-akait** [comment]
Can you reformat your issue, maybe create reproducible test repo on github, because it is not readable thank you
